cpp/src/gandiva/like_holder_test.cc

```diff
@@ -84,6 +84,16 @@ TEST_F(TestLikeHolder, TestRegexEscape) {
   EXPECT_EQ(res, "%hello_abc.def#");
 }
 
+TEST_F(TestLikeHolder, TestDot) {
+  std::shared_ptr<LikeHolder> like_holder;
+
+  auto status = LikeHolder::Make("abc.", &like_holder);
+  EXPECT_EQ(status.ok(), true) << status.message();
```

Comment:
`EXPECT_TRUE(status.ok()) <<...`

Reply:
while i agree it might be more succinct, if you dont feel strongly about, i would prefer to leave as is to remain consistent with the rest of the file (since they both mean the same anyways)

i will focus on the CI failure - looks like this entire unit test was disabled by mistake for a while and on re-introducing we are getting new errors.